README.md

```diff
@@ -99,45 +99,42 @@ Here are two common scenarios involving a user and a participating third-party w
 
 The user initiates the registration process by entering their email address and a password into the website. The use of Recaptchas on most websites rules out a full-fledged _Idem_ registration, although _Idem_ could be used to fill the email address and generate a password on behalf of the user. From here, the user can use _Idem_ to complete the verification process by allowing _Idem_ to supply the claims the website requires.
 
-### 2. Existing (to the website) user
+### 2. Existing (to the website) non-verified user
 
-An existing user can log into the website by scanning the QR code (using _Idem_) the website displays, Recaptcha allowing. Otherwise, the user can simply log in manually and enjoy the benefits of the website, having already presented their credentials using _Idem_.
-
-Note that for MVP purposes (see below), the claims required will all already have been saved to the user's device during their initial _Idem_ set-up. _Idem_ will also be able to update any claims the website requires, such as date of birth, name, or address, or provide extra credentials if the user wanted to access a new part of the site.
+An existing user can log into the website, given the user is using the same email address as their claim in IDEM, the user will receive an alert stating "Would you like to use IDEM to verify with the website?" If the user accepts this alert they will be verified on the platform once their claims have been verified.
 
 ## Verification Workflow Diagram
 
 The flowchart below is a verification workflow diagram for third-party developers to integrate a website (such as an exchange) with _Idem_. It works as follows:
 
 1. A user creates account on the website, typically with a email and password.
-2. When the user signs up, an account is created.
-3. The new user may then be assed to supply more information to meet KYC obligations. This could be meta data such as Full Name, Date of Birth and Phyiscal address. These are what Idem referrers to as _Claims_. To capture this data, the website can either
-4. - Get the user to fill out a form
-5. - Present a self documenting QR code requesting the claims required, and the callback URL for the website. This is similar to a PayPal IPN or an OAuth2 callback URL.
-6. The user is asked to confirm they're happy to proceed with sharing those specific claims to the website.
-7. IDEM then posts the claims in DID format to the websites specified callback URL, which it can then add to the users profile.
+2. The new user may then be assed to supply more information to meet KYC obligations. This could be meta data such as Full Name, Date of Birth and Phyiscal address. These are what Idem referrers to as _Claims_. To capture this data, the website can either
```

Comment:
```suggestion
2. The new user may then be assed to supply more information to meet KYC obligations. This could be meta data such as Full Name, Date of Birth and Phyiscal address. These are what Idem referrers to as _Claims_. To capture this data, the website can either.
```